src/cycax/cycad/cycad_side.py

```diff
@@ -0,0 +1,463 @@
+        horizontal=True,
+        inner: bool = True,
+        external_only: bool = False,
+    ):
+        """
+        This allows a slot cut out to be cut into the specified side.
+        Args:
+            pos (tuple[float, float]): The (x,y) coordinates of the slot cut out.
+            length (float): The length of the slot as viewed from the specified side.
+            width (float): The width of the slot as viewed from the specified side.
+            depth (float, optional): The depth of the slot as viewed from the specified side. If not specified it will cut the slot all the way through the surface. Defaults to None.
+            horizontal (bool, optional): Slots can either run verticall or horizontally. Defaults to horizontal.
+            inner (bool, optional): This is specified that the slot is internal and will not be coppied onto other surface. Defaults to True.
+            external_only (bool, optional): This is specified that the slot will only be tranferred onto other surfaces and will not be drilled into main object. Defaults to False.
+        """
+        _depth = self._depth_check(depth)
+        _location_tupple = self._location_calc(pos=pos, sink=0.0)
+        _box_dimentions = self._box_size_calc(width=width, length=length, depth=_depth)
+        self._parent.make_slot(
+            x=_location_tupple[0],
+            y=_location_tupple[1],
+            z=_location_tupple[2],
+            side=self.name,
+            x_size=_box_dimentions[0],
+            y_size=_box_dimentions[1],
+            z_size=_box_dimentions[2],
+            horizontal=horizontal,
+            inner=inner,
+            external_only=external_only,
+        )
+
+    def _depth_check(self, val: float):
+        """
+        This is used to check the depth of an object. This method is just used for error catching.
+
+        Args:
+            val (float): Depth of object.
+
+        Raises:
+            ValueError: The side specidfied does not have a depth check method.
+        """
+        raise ValueError("_depth_check is Not implimented on" + self.name)
+
+
+class LeftSide(CycaxSide):
+    name = "LEFT"
+
+    def _location_calc(self, pos: tuple[float, float], sink: float = 0.0) -> tuple[float, float, float]:
+        """
+        location is calculated for the (x, y) plane using two values and a side.
+
+        Args:
+            pos (tuple[float, float]): (x, y) location of an object which is used to find the location with the side.
+            sink (float, optional): The object can be sunk bellow the surface of the specified side to make a pocket. Defaults to 0.0.
+
+        Returns:
+            tuple[float, float, float]: (x, y, z) location of an object.
+        """
+
+        temp_x = self._parent.x_min + sink
+        temp_y = pos[0]
+        temp_z = pos[1]
+
+        return temp_x, temp_y, temp_z
+
+    def _box_size_calc(self, width: float, length: float, depth: float) -> tuple[float, float, float]:
```

Comment:
Please also define _box_size_calc in the parent class.

FYI in the future. You only have to comment the method in the parent class (CycaxSide), then you just say "See CycaxSide._box_size_calc for documentation.". Thats only applicable for private methods.